SocketRocket/ProxyConnect.h

```diff
@@ -0,0 +1,6 @@
+#import <Foundation/Foundation.h>
```

Comment:
This is missing the copyright declaration, which is required for all our source files.
Please copy it from `SRError.h`